src/routes/plans/[id]/+page.svelte

```diff
@@ -170,4 +176,43 @@
     on:changeColumnSizes={changeColumnSizes}
     on:changeRowSizes={changeRowSizes}
   />
+  <Console>
+    <svelte:fragment slot="console-tabs">
+      <div class="console-tabs">
+        <div>
+          <ConsoleTab numberOfErrors={$allErrors?.length}>All</ConsoleTab>
+        </div>
+        <div class="separator">|</div>
+        <div class="grouped-error-tabs">
+          <ConsoleTab numberOfErrors={$simulationDatasetErrors?.length}><PlayIcon /></ConsoleTab>
+          <ConsoleTab numberOfErrors={$schedulingErrors?.length}><BookIcon /></ConsoleTab>
```

Comment:
Additionally can we use the same iconography we use in the top nav bar? So for simulation it's the gear, and for scheduling it's the calendar. It would be nice to have custom error icons for these sections at some point.
CC @lklyne

Reply:
Good call. I was going off of the design, which includes changes to all the icons. Is there a ticket for updating the icons to match the design?

Reply:
Nope not yet. I'll make one.

Reply:
https://jira.jpl.nasa.gov/browse/AERIE-2147